Doesn't install on Windows 10

I have installed successfully on my macs, but on a Windows Surface Laptop I get when running the first command:
mvn io.quarkus:quarkus-maven-plugin:0.15.0:create \
>>     -DprojectGroupId=org.acme \
>>     -DprojectArtifactId=getting-started \
>>     -DclassName="org.acme.quickstart.GreetingResource" \
>>     -Dpath="/hello"

….

[INFO] using-keycloak ..................................... SKIPPED
[INFO] quickstart-projects ................................ FAILURE [ 11.386 s]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  12.364 s
[INFO] Finished at: 2019-05-24T10:05:09+01:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal io.quarkus:quarkus-maven-plugin:0.15.0:create (default-cli) on project quickstart-projects: Execution default-cli of goal io.quarkus:quarkus-maven-plugin:0.15.0:create failed.: NullPointerException -> [Help 1]
(Describe the problem clearly and concisely.)

**Expected behavior**
(Describe the expected behavior clearly and concisely.)


**Configuration**
Windows 10 Surface Laptop (gen 1) 16GB RAM
Windows 10 Pro v1803
java version "1.8.0_211"
Java(TM) SE Runtime Environment (build 1.8.0_211-b12)
Java HotSpot(TM) 64-Bit Server VM (build 25.211-b12, mixed mode)
Latest Quarkus
(also tried with Java 1.12 same issue)

Sure, here you go:
Apache Maven 3.6.1 (d66c9c0b3152b2e69ee9bac180bb8fcc8e6af555; 2019-04-04T20:00:29+01:00)


[INFO] quickstart-projects ................................ FAILURE [  2.101 s]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  3.026 s
[INFO] Finished at: 2019-05-24T16:00:02+01:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal io.quarkus:quarkus-maven-plugin:0.15.0:create (default-cli) on project quickstart-projects: Execution default-cli of goal io.quarkus:quarkus-maven-plugin:0.15.0:create failed.: NullPointerException -> [Help 1]
org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal io.quarkus:quarkus-maven-plugin:0.15.0:create (default-cli) on project quickstart-projects: Execution default-cli of goal io.quarkus:quarkus-maven-plugin:0.15.0:create failed.
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:215)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:156)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:148)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:81)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:56)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:128)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:305)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:192)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:105)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:956)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:288)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:192)
    at sun.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:498)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:282)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:225)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:406)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:347)
Caused by: org.apache.maven.plugin.PluginExecutionException: Execution default-cli of goal io.quarkus:quarkus-maven-plugin:0.15.0:create failed.
    at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo (DefaultBuildPluginManager.java:148)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:210)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:156)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:148)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:81)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:56)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:128)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:305)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:192)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:105)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:956)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:288)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:192)
    at sun.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:498)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:282)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:225)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:406)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:347)
Caused by: java.lang.NullPointerException
    at io.quarkus.cli.commands.CreateProject.hasPlugin (CreateProject.java:212)
    at io.quarkus.cli.commands.CreateProject.addMainPluginConfig (CreateProject.java:194)
    at io.quarkus.cli.commands.CreateProject.doCreateProject (CreateProject.java:133)
    at io.quarkus.maven.CreateProjectMojo.execute (CreateProjectMojo.java:156)
    at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo (DefaultBuildPluginManager.java:137)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:210)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:156)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:148)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:81)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:56)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:128)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:305)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:192)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:105)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:956)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:288)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:192)
    at sun.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:498)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:282)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:225)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:406)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:347)
[ERROR]
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/PluginExecutionException
[ERROR]
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <goals> -rf :quickstart-projects
-DprojectGroupId=org : The term '-DprojectGroupId=org' is not recognized as the name of a cmdlet, function, script
file, or operable program. Check the spelling of the name, or if a path was included, verify that the path is correct
and try again.
At line:2 char:5
+     -DprojectGroupId=org.acme \
+     ~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (-DprojectGroupId=org:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException

-DprojectArtifactId=getting-started : The term '-DprojectArtifactId=getting-started' is not recognized as the name of
a cmdlet, function, script file, or operable program. Check the spelling of the name, or if a path was included,
verify that the path is correct and try again.
At line:3 char:5
+     -DprojectArtifactId=getting-started \
+     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (-DprojectArtifactId=getting-started:String) [], CommandNotFoundExceptio
   n
    + FullyQualifiedErrorId : CommandNotFoundException

-DclassName=org.acme.quickstart.GreetingResource : The term '-DclassName=org.acme.quickstart.GreetingResource' is not
recognized as the name of a cmdlet, function, script file, or operable program. Check the spelling of the name, or if
a path was included, verify that the path is correct and try again.
At line:4 char:5
+     -DclassName="org.acme.quickstart.GreetingResource" \
+     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (-DclassName=org...reetingResource:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException

-Dpath=/hello : The term '-Dpath=/hello' is not recognized as the name of a cmdlet, function, script file, or operable
program. Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
At line:5 char:5
+     -Dpath="/hello"
+     ~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (-Dpath=/hello:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException

--- a/devtools/common/src/main/java/io/quarkus/cli/commands/CreateProject.java
+++ b/devtools/common/src/main/java/io/quarkus/cli/commands/CreateProject.java
@@ -192,6 +192,7 @@ public class CreateProject {
 
     private void addMainPluginConfig(Model model) {
         if (!hasPlugin(model)) {
+            Build build = createBuildSectionIfRequired(model);
             Plugin plugin = plugin(getPluginGroupId(), getPluginArtifactId(), QUARKUS_VERSION_PROPERTY);
             if (isParentPom(model)) {
                 addPluginManagementSection(model, plugin);
@@ -201,32 +202,31 @@ public class CreateProject {
             PluginExecution pluginExec = new PluginExecution();
             pluginExec.addGoal("build");
             plugin.addExecution(pluginExec);
-            Build build = createBuildSectionIfRequired(model);
             build.getPlugins().add(plugin);
         }
     }
 
     private boolean hasPlugin(final Model model) {
         List<Plugin> plugins = null;
-        if (isParentPom(model)) {
-            final PluginManagement management = model.getBuild().getPluginManagement();
-            if (management != null) {
-                plugins = management.getPlugins();
-            }
-        } else {
-            final Build build = model.getBuild();
-            if (build != null) {
+        final Build build = model.getBuild();
+        if (build != null) {
+            if (isParentPom(model)) {
+                final PluginManagement management = build.getPluginManagement();
+                if (management != null) {
+                    plugins = management.getPlugins();
+                }
+            } else {
                 plugins = build.getPlugins();
             }
         }
-        return plugins != null && model.getBuild().getPlugins()
+        return plugins != null && build.getPlugins()
                 .stream()
                 .anyMatch(p -> p.getGroupId().equalsIgnoreCase(getPluginGroupId()) &&
                         p.getArtifactId().equalsIgnoreCase(getPluginArtifactId()));
     }
 
     private void addPluginManagementSection(Model model, Plugin plugin) {
-        if (model.getBuild().getPluginManagement() != null) {
+        if (model.getBuild() != null && model.getBuild().getPluginManagement() != null) {
             if (model.getBuild().getPluginManagement().getPlugins() == null) {
                 model.getBuild().getPluginManagement().setPlugins(new ArrayList<>());
             }

--- a/extensions/smallrye-context-propagation/deployment/src/test/java/io/quarkus/context/test/TransactionalBean.java
+++ b/extensions/smallrye-context-propagation/deployment/src/test/java/io/quarkus/context/test/TransactionalBean.java
@@ -2,16 +2,12 @@ package io.quarkus.context.test;
 
 import javax.enterprise.context.ApplicationScoped;
 import javax.inject.Inject;
-import javax.transaction.SystemException;
 import javax.transaction.TransactionManager;
 import javax.transaction.Transactional;
 import javax.transaction.Transactional.TxType;
 
-import org.eclipse.microprofile.reactive.streams.operators.ReactiveStreams;
 import org.junit.jupiter.api.Assertions;
-import org.reactivestreams.Publisher;
 
-import io.quarkus.hibernate.orm.panache.Panache;
 import io.reactivex.Flowable;
 import io.reactivex.Single;
 

--- a/extensions/smallrye-context-propagation/runtime/src/main/java/io/quarkus/smallrye/context/runtime/SmallRyeContextPropagationTemplate.java
+++ b/extensions/smallrye-context-propagation/runtime/src/main/java/io/quarkus/smallrye/context/runtime/SmallRyeContextPropagationTemplate.java
@@ -3,7 +3,6 @@ package io.quarkus.smallrye.context.runtime;
 import java.util.List;
 import java.util.concurrent.ExecutorService;
 
-import org.eclipse.microprofile.context.spi.ContextManager;
 import org.eclipse.microprofile.context.spi.ContextManagerExtension;
 import org.eclipse.microprofile.context.spi.ContextManagerProvider;
 import org.eclipse.microprofile.context.spi.ThreadContextProvider;

--- a/integration-tests/main/src/test/java/io/quarkus/it/main/ContextPropagationTestCase.java
+++ b/integration-tests/main/src/test/java/io/quarkus/it/main/ContextPropagationTestCase.java
@@ -18,16 +18,8 @@ package io.quarkus.it.main;
 
 import static org.hamcrest.Matchers.is;
 
-import java.io.ByteArrayOutputStream;
-import java.io.InputStream;
-import java.net.URL;
-import java.net.URLConnection;
-import java.nio.charset.StandardCharsets;
-
-import org.junit.jupiter.api.Assertions;
 import org.junit.jupiter.api.Test;
 
-import io.quarkus.test.common.http.TestHTTPResource;
 import io.quarkus.test.junit.QuarkusTest;
 import io.restassured.RestAssured;
 

